### ISSUE TITLE
Add a search method, to compare a non-corpus document to the corpus

### ISSUE THREAD

**Issue by jpmckinney** [jpmckinney/tf-idf-similarity#6]

**Comment by ParthKolekar** [comment]
This would be great to have.

**Comment by neymarsabin** [comment]
Any updates on above mentioned idea?

**Comment by jpmckinney** [comment]
Sorry, no update - I'm not actively using the gem, but happy to merge a PR :)

**Comment by neymarsabin** [comment]
Are you implementing cosine similarity in this gem for the last computation ?

**Comment by jpmckinney** [comment]
What last computation are you referring to?

**Comment by neymarsabin** [comment]
Finding similarity after you find out the tf-idf values and the matrix. Doesn't the code below find the document similarity between document1 and document2 using cosine similarity method?  
`matrix[model.document_index(document1), model.document_index(document2)]`

**Comment by jpmckinney** [comment]
Yes, indeed. That's how to do it if the document is a part of the corpus.

**Comment by neymarsabin** [comment]
@jpmckinney Thanks for the information...
